internal/core/api.rs

```diff
@@ -279,6 +279,26 @@ impl<'a> core::fmt::Debug for GraphicsAPI<'a> {
     }
 }
 
+/// This enum specifies which OpenGL API should be used.
+#[derive(Debug, Clone, PartialEq)]
+pub enum OpenGLApi {
```

Comment:
Should this be `OpenGLAPI`? I noticed this repo tends to prefer capitalizing initialisms, but then it's hard to read if there are two next to each other (what's a "GLAPI"?).

Reply:
Yes, I think it should be API. We might revisit this before the next release as we do another API (🤣) review on the changes since the previous release.

Reply:
Force pushed a reword to `API` along with my other changes.